eng/Version.Details.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.3.20179.1">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>1d4a35ec9ecb9bcd8fb26150fb6da3deb4aab795</Sha>
+    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.3-runtime.20179.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
I'm pretty sure this repo doesn't need CPD attributes for anything except the dotnet/roslyn dependency. @wtgodbe @mmitche is that correct i.e. is there any reason for CPD links between packages coming from the same (dotnet/runtime) repo?

Reply:
We still need the CPDs while runtime flows through extensions to this repo (which is still the case). Once we ingest runtime directly we can remove the CPDs

Reply:
Huh? The words `dotnet/extensions` don't appear anywhere in this file; everything comes from arcade, roslyn or runtime.

Reply:
Oh, I see, I had this backwards - in that case this PR also depends on adding a 'runtime -> aspnetcore-tooling' subscription in Darc

Reply:
yes

Reply:
I wasn't planning on updating darc but I suppose I could.